.github/actions/k-bench/README.md

```diff
@@ -0,0 +1,157 @@
+# K-Bench
+
+## Continous Benchmarking
+The K-Bench action runs K-Bench benchmarks on Constellation clusters.
+The benchmark suite records storage, network, and Kubernetes API benchmarks.
+
+After testing, the action compares the results of the benchmarks to previous results of Constellation on the same cloud provider. That way, it is possible to evaluate performance progression throughout the development.
+
+The data of previous benchmarks is stored in the public S3 bucket.
```

Comment:
Mh, personally my idea was to deprecate the old public S3 bucket slowly. Do we actually publish this data somewhere publicly?

I wonder if we shouldn't just create a private bucket for various artifacts we use internally.